.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+**Before merging:**
+
+- [ ] Copy the latest benchmark results into the `README.md` and update this PR
```

Comment:
Should this say "if they have changed beyond the expected slight noise"? Or similar?

Should this checklist item link to the Github Action results area or guide the contributor to know where to look for the results?

Can we guide them more specifically to `README.md` in the section *Section Name* or something?

Reply:
> Should this say "if they have changed beyond the expected slight noise"? Or similar?

I'm torn. I agree it may not be that useful but I worry about us missing changes because we're used to not changing this.

> Should this checklist item link to the Github Action results area or guide the contributor to know where to look for the results?

Sure. I'm not sure how to link to the results since I think the PR number will be in the URL. Do you know of a way?

> Can we guide them more specifically to README.md in the section Section Name or something?

Sure.

Reply:
> I'm torn. I agree it may not be that useful but I worry about us missing changes because we're used to not changing this.

I actually like the idea of updating it each time too. I think it gives complete objectivity to our benchmarks as opposed to introducing any subjectivity or potential confusion for any external folks hoping to contribute to this gem who might not know what "expected noise" would be.

Reply:
Ok! I'm convinced by both reasonings